pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -430,6 +462,17 @@ func (r *ReconcileUserSignup) setStatusComplete(userSignup *toolchainv1alpha1.Us
 		})
 }
 
+func (r *ReconcileUserSignup) setStatusDeactivated(userSignup *toolchainv1alpha1.UserSignup, message string) error {
+	return r.updateStatusConditions(
+		userSignup,
+		toolchainv1alpha1.Condition{
+			Type:    toolchainv1alpha1.UserSignupComplete,
+			Status:  corev1.ConditionTrue,
```

Comment:
Do you think we should set the Complete condition to `true` for deactivated signups?
I think it depends on the logic for the re-signup in the reg service we want to implement...

Reply:
I was wondering about this, however it seems to me that the `Complete` condition should be set to `true` once the resources reach the desired state based on their specification.  In the case of a new user signup, we set complete to true once the MUR is created, for a deactivation, it seems correct that we would set it to true once the MUR is deleted, as there is no longer any work remaining.  Can you think of an alternative way to approach this?

Reply:
Yeah I see your point. It makes sense. We could just check the reason in reg-service when checking the signup/mur statuses. If Completed is true but the reason is Deactivated then we know that user has been deactivated.